products/bash/1-click-pre-validation.sh

```diff
@@ -74,6 +74,11 @@ if [[ -z "${namespace// }" ]]; then
   missingParams="true"
 fi
 
+if [[ -z "${namespace// }" ]]; then
```

Comment:
You have a duplicate check for the namespace here 
![image](https://user-images.githubusercontent.com/67965358/98118259-e0cd0580-1ea2-11eb-865a-92da846489b7.png)

Reply:
Done